### ISSUE TITLE
Interface not defined

### ISSUE THREAD

**Issue by SterredenBreeijen** [AmpersandTarski/Ampersand#1036]
#### Version of ampersand that was used 
v3.17.4 [master:fd90ea3f1]

#### What I expected
For some testing, I tried building this interface: 
```
-- [INTERFACE]
INTERFACE "Overview": "_SESSION"[SESSION]   
    TABS [ "Party": V[SESSION * Party]  cRud BOX [
            "Party": I[Party]
            , "Guardian of": isGuardianOf
            , "responsible for": isResponsibleFor
        ]   
   ]
```

#### What happened instead
I got an error that the interface "Guardianship" is not defined, which is nowhere in my code. 
![afbeelding](https://user-images.githubusercontent.com/49937913/71712358-25ac8980-2e05-11ea-90f1-c218e6ee06ad.png)

#### Steps to reproduce
I tried building by interface by the `--proto`.

**Comment by hanjoosten** [comment]
Hi Sterre, could you supply your .adl script, so I can reproduce this?

**Comment by SterredenBreeijen** [comment]
Hi Han,

I deleted my rules so you won't get errors for those. 
I asked Rieks yesterday as well, he got the error 'Interface Administrator' is not defined, using the same script. 

```
CONCEPT Party "People and organizations"
CONCEPT Actor "an actor can be used by a party to do the issuing and validation on their behalf, i.e. a person or machine"
CONCEPT Agent "equipment, used in business transactions, i.e. app on phone, webserver, ..."

CONCEPT Jurisdiction "Sommige varianten van guardianship zijn alleen binnen bepaalde context geldig zoals bewindvoering (alleen in NL), of de Mya usecase (binnen de hekken van haar kamp)"


partyActsVia :: Party * Actor         -- the party acts via an actor, example: TNO acts via Sterre as employee in a project
actorUsesAgent :: Actor * Agent       -- the actor uses an agent, example: Sterre uses app on phone

isGuardianOf :: Guardian * Dependent  -- party (the guardian) that acts on behalf of another party (the dependent), I act on behalf of my mother
CLASSIFY Dependent ISA Party          -- the dependent is the person who cannot do all business transactions himself, someone else will take care of this
CLASSIFY Guardian ISA Party           -- the guardian is the person who helps a dependent do certain business transactions, acts on behalf of the dependent (no impersonation)

partyIsInJurisdiction :: Party * Jurisdiction
isResponsibleFor :: Party * Party [PROP]


-- [INTERFACE]
INTERFACE "Overview": "_SESSION"[SESSION]   
    TABS [ "Party": V[SESSION * Party]  cRud BOX [
            "Party": I[Party]
            , "Guardian of": isGuardianOf
            , "responsible for": isResponsibleFor
        ]   
   ]
```

**Comment by hanjoosten** [comment]
Thanks for the script. It contains no errors, I can compile it and I can run it:

![image](https://user-images.githubusercontent.com/1159672/71714255-614b5180-2e0d-11ea-810f-5bb8a03b04c1.png)


However, during compilation I did not have (and thus not used) special configuration, that might be present in your environment. 
Are there any subdirectories in the directory where you have your script? Special interest are subdirectories with names like `customizations` and/or `templates`.

**Comment by SterredenBreeijen** [comment]
Hi Han,
I just talked to Michiel, apparently I had some databases still living somewhere in phpMyAdmin. After deleting them, the interface is built.

**Comment by hanjoosten** [comment]
Ok. Thanks for the comment. I'll close this issue. Feel free to reopen if you disagree.

**Comment by SterredenBreeijen** [comment]
Hi Han,
Could you reopen this issue and assign Michiel? He wants change some things so another error appears or prevent the error. I don't have the rights for this.

**Comment by hanjoosten** [comment]
@Michiel-s

**Comment by Michiel-s** [comment]
This issue is a duplicate of #1005 and is fixed in [release v1.4.0 of the prototype framework](https://github.com/AmpersandTarski/prototype/releases/tag/v1.4.0). A new release of Ampersand v3.x (>3.17.4) is needed to update to v1.4.0 of the prototype framework.

This is work in progress, see: https://github.com/AmpersandTarski/Ampersand/tree/master-v3x
